src/main/java/io/netty/incubator/codec/http3/Http3ClientConnectionHandler.java

```diff
@@ -41,11 +41,13 @@ public Http3ClientConnectionHandler() {
      *                                              {@code null} if no special handling should be done.
      * @param localSettings                         the local {@link Http3SettingsFrame} that should be sent to the
      *                                             remote peer or {@code null} if the default settings should be used.
+     * @param disableQPACKDynamicTable              If QPACK dynamic table should be disabled.
```

Comment:
nit: we also use camel-case for `Qpack` and `Http3` in the classnames so I think we should do the same in the parameter name.